cpp/models/ode_secirvvs/parameters_io.h

```diff
@@ -0,0 +1,1164 @@
+                                                           age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::CarrierFullyImmune +
+                                                           age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::Infected + age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::InfectedPartiallyImmune +
+                                                           age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::InfectedFullyImmune +
+                                                           age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::Hospitalized + age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::HospitalizedPartiallyImmune +
+                                                           age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::HospitalizedFullyImmune +
+                                                           age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::ICU + age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::ICUPartiallyImmune +
+                                                           age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::ICUFullyImmune + age_group_offset) +
+                             extrapolated_rki[county][day]((size_t)InfectionState::Dead + age_group_offset));
+                    }
+                }
+                else {
+                    log_warning("No population data available for region " + std::to_string(county) +
+                                ". Population data has not been set.");
+                }
+            }
+            log_info("extrapolated real data for date: {}-{}-{}", date.day, date.month, date.year);
+            date = offset_date_by_days(date, 1);
+        }
+        /* end: similar functionality in set_rki_data(), here only for vector of TimeSeries */
+        auto num_groups = (int)(size_t)model[0].parameters.get_num_groups();
+        BOOST_OUTCOME_TRY(save_result(extrapolated_rki, region, num_groups, path_join(results_dir, "Results_rki.h5")));
+
+        auto extrapolated_rki_data_sum = sum_nodes(std::vector<std::vector<TimeSeries<double>>>{extrapolated_rki});
+        BOOST_OUTCOME_TRY(save_result({extrapolated_rki_data_sum[0][0]}, {0}, num_groups,
+                                      path_join(results_dir, "Results_rki_sum.h5")));
+
+        return success();
+    }
+
+    template <class Model>
+    IOResult<void> read_input_data_county(std::vector<Model>& model, Date date, const std::vector<int>& county,
```

Comment:
you were right, replaced `< num_days` with `<= num_days` so that num_days + 1 time points are generated as simulation also creates num_days + 1 time points ( 1 at t0 + 1 per day of simulation).